composer/workflows/integrated_example.py

```diff
@@ -0,0 +1,140 @@
+"""An example Composer workflow integrating BigQuery, Dataflow, and GCS.
+   This workflow runs a Dataflow job on a CSV file, stores the output dataset in
+   a BigQuery table, performs a query on the dataset, and writes the query results
+   back to a GCS bucket.
+
+   Before starting, configure the `gcp_project` and `gcs_bucket` environment
+   variables. The `gcs_ubkcet` value can be the same Composer bucket in which
+   you upload this DAG.
+
+   gcloud composer environments run composer-exercise variables -- \
+       --set gcp_project [your project id]
+
+   gcloud composer environments run composer-exercise variables -- \
+       --set gcs_bucket [your gcs bucket]
+
+   Also, be sure that you've enabled the Dataflow API for your project.
+"""
+
+from datetime import datetime, timedelta
+from airflow import DAG
+from airflow.contrib.operators.bigquery_operator import BigQueryOperator
+from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
+from airflow.contrib.operators.dataflow_operator import DataFlowPythonOperator
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
+
+YESTERDAY = datetime.combine(datetime.today() - timedelta(days=1),
+                             datetime.min.time())
+
+BUCKET = Variable.get('gcs_bucket')
+BQ_DATASET_NAME = 'example_dataset'
+BQ_RESULT_TABLE = '{0}.{1}'.format(BQ_DATASET_NAME, 'results')
+
+DEFAULT_ARGS = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': YESTERDAY,
+    # If you'd like to test out enabling e-mail notifications on DAG
+    # retries/failure, configure your environment to support email by
+    # setting the SENDGRID variables and modifying the email fields below.
+    'email': ['email@foobar.com'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=5),
+    'dataflow_default_options': {
+        'project': Variable.get('gcp_project'),
+        'stagingLocation': '{0}/{1}'.format(BUCKET, 'staging/'),
+        'tempLocation': '{0}/{1}'.format(BUCKET, 'temp/'),
+    }
+}
+
+with DAG('integrated_example', default_args=DEFAULT_ARGS) as dag:
+  # We need to create a BigQuery dataset to store the output from Dataflow.
```

Comment:
Per PEP-8 style, Use 4 spaces not 2 on public samples. You can run the `flake8` command to lint this file.

Nit on the comment, in general don't use "we" in docs. Prefer: "Create a BigQuery dataset to store output ..." or possibly "This command creates ..."